core/src/main/java/org/apache/iceberg/actions/RewritePositionDeletesGroup.java

```diff
@@ -50,16 +53,16 @@ public RewritePositionDeletesGroup(
       FileGroupInfo info,
       List<PositionDeletesScanTask> tasks,
       long writeMaxFileSize,
-      long splitSize,
+      long inputSplitSize,
```

Comment:
Same as above, not sure we should do a rename here

Reply:
I have forgot to rename these in my previous PR. During the review we changed from splitSize to inputSplitSize to highlight that this is for the input. We did the rename in the parent class (RewriteGroupBase), but I didn't do it here. Since this is a new class introduced by my previous PR, I felt that we should keep the naming consistent.